clang_delta/CMakeLists.txt

```diff
@@ -352,6 +354,12 @@ foreach(file IN LISTS SOURCE_FILES)
   configure_one_file(${file})
 endforeach()
 
+if (MSVC)
```

Comment:
Is this one a necessary change?

Reply:
For me it is not compiling on MSVC without \bigobj

Reply:
I see, so keep this commit and squash it to 2 commits.